documentation/meta/project_boards/index.md

```diff
@@ -0,0 +1,25 @@
+# Project boards
+
+Openverse project management consists of three project boards,
```

Comment:
```suggestion
Openverse project management consists of four project boards,
```

Reply:
Haha, that was so stupid. Thanks for catching it. Fixed in b54362b.